backend/main.py

```
@@ -0,0 +1,227 @@
+            "title": task.title,
+            "description": task.description,
+            "priority": task.priority,
+            "due_date": task.due_date,
+            "status": "pending",
+            "created_at": datetime.now()
+        }
+        tasks.append(new_task)
+        return {"task": new_task, "message": "Task created successfully"}
+        
+    except Exception as e:
+        raise HTTPException(status_code=500, detail=f"Error creating task: {str(e)}")
```

Ruff (0.12.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)